### ISSUE TITLE
Sybil Attacker Report

### ISSUE THREAD

**Issue by shdwy** [hop-protocol/hop-airdrop#388]
# Related Addresses

```
0x4a2ff57467db1290cf32cf106c96987d5f4eca1a
0xbd2f62fb5a4287549db9a1af9278fb78cbbed6a8
0x4eae801c4038cb5fd4e428febc9b0f357413591b
0x7c59fe436b97de2b6df05d696a2d8562506d1c74
0x2e4601bd54a44ff8bf44097c6e49dde816cbcffe
0xc7620ec74609a00366e6d7aabea8310502713c63
0x7685598d7da42507158014db0842352c1280457f
0x71d8c525fea9ca23ed6882db1f868750e15f0ab0
0x236d06dc4d853bbf7234d457881ed6634b7eaafb
0xb86de6562a37c805dc43cdc12451f6823988e6c6
```

# Reasoning

All addresses receive funds (on Polygon) from the previous address on the chain, they interact with the same (or similar) set of contracts (e.g. they all do a 100 Matic swap on Paraswap on Aug 21, 2021) and then transfer funds to the next address on the chain, all in a 24h window.

# Methodology

Same as Issue #354 

# Rewards Address

0xD47CB1835EB36e4A108C33E446449192426Fc160

**Comment by shanefontaine** [comment]
Thank you for your report @shdwy. We have verified that the addresses in this report are Sybil attackers.

The report included 10 eligible addresses as Sybil attackers which means you are eligible for 4017.386865127362551328 HOP! When Hop DAO is live, we will make a proposal for this reward — subject to a 1 year lockup, as mentioned in the [original Mirror post](https://hop.mirror.xyz/AI5fOUR0X_l0mktShDOx3mwr-hsB24gp8GvTWtS-MBc).


The qualified addresses are as follows:

```
0x4a2ff57467db1290cf32cf106c96987d5f4eca1a
0xbd2f62fb5a4287549db9a1af9278fb78cbbed6a8
0x4eae801c4038cb5fd4e428febc9b0f357413591b
0x7c59fe436b97de2b6df05d696a2d8562506d1c74
0x2e4601bd54a44ff8bf44097c6e49dde816cbcffe
0xc7620ec74609a00366e6d7aabea8310502713c63
0x7685598d7da42507158014db0842352c1280457f
0x71d8c525fea9ca23ed6882db1f868750e15f0ab0
0x236d06dc4d853bbf7234d457881ed6634b7eaafb
0xb86de6562a37c805dc43cdc12451f6823988e6c6
```

**Comment by forweb3** [comment]
sir ，One of my, our report is finding that the rule has to be a jump to survive. However, the successor has completed more than 10 user tracking records, and there is no record of him. I cancelled his reward. And this is what I found so am I not rewarded accordingly? This address has not interacted with hop. And he did not provide other information that he interacted. I report all his reports.
0xD47CB1835EB36e4A108C33E446449192426Fc160 This is a brand new empty address, why can't he provide the address he interacts with, because he is the real empty hunter. He circumvented it with better technical isolation. So here's my reasoning, I want 25% of his reward to go to me and then the remaining 75% to all eligible real users. Just like our rules.
@shanefontaine
